soccer/planning/DirectPathPlanner.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "planning/Path.hpp"
+#include "planning/PathPlanner.hpp"
+
+namespace Planning {
+    class DirectPathPlanner : public PathPlanner {
```

Comment:
Ok, well if it's not in use now, can we remove it from this PR and put it on a separate branch?  I'd rather not have bits and pieces of potential new implementations floating around :)